docs/develop/enable-nested-app-authentication-in-your-add-in.md

```diff
@@ -18,6 +17,12 @@ You can use the MSAL.js library (version 3.11 and later) with nested app authent
 
 > [!IMPORTANT]
 > Nested app authentication (NAA) is currently in preview. To try this feature, join the Microsoft 365 Insider Program (https://insider.microsoft365.com/join) and choose the Beta Channel. Don't use NAA in production add-ins. We invite you to try out NAA in test or development environments and welcome feedback on your experience through GitHub (see the **Feedback** section at the end of this page).
+> NAA is supported in the following builds.
+>
+> - Word, Excel, and PowerPoint on Windows build 16.0.17531.20000 or later.
+> - Word, Excel, and PowerPoint on Mac build 16.85.24040319 or later.
+> - Outlook on Windows build 16.0.17531.20000 or later.
+> - Outlook on Mac build 16.85.24040319 or later.
```

Comment:
Why list Outlook separately? The build numbers are the same as for WXP.

Reply:
Please remove period on bulleted items. You only use when items are sentences.

Reply:
fixed, thanks!